content/conditions/cold/main-content.md

```diff
@@ -58,3 +58,36 @@ There’s little evidence that supplements (such as vitamin c, zinc, echinacea o
 
 GPs don’t usually recommend antibiotics for colds because they won’t relieve your symptoms or speed up your recovery.
 Antibiotics are only effective against bacterial infections and colds are caused by viruses.
+
+!!! alert
+  ## See someone urgently if  
```

Comment:
Check with content about the use of "someone". Feels like we should be less vague, like "a GP now"